consensus/view_change.go

```diff
@@ -426,7 +426,10 @@ func (consensus *Consensus) onNewView(msg *msg_pb.Message) {
 		consensus.getLogger().Warn().Err(err).Msg("[onNewView] Unable to Parse NewView Message")
 		return
 	}
-
+	if consensus.blockNum < recvMsg.BlockNum {
```

Comment:
Isn't the node going to auto-sync itself by the periodic "out of sync" check? Do we really need this trigger too?

Reply:
Umm, you are right. Closing